Dockerfile

```diff
@@ -9,11 +9,14 @@ RUN apk add --no-cache \
     py2-pip \
     mongodb-tools \
     curl
-    
+
+# install awscli
 RUN pip install awscli
-RUN curl https://storage.googleapis.com/pub/gsutil.tar.gz | tar xz -C $HOME
-RUN echo "export PATH=${PATH}:$HOME/gsutil" >> ~/.bashrc
 
+# install gcloud (also gsutil)
+ARG GCLOUD_URL=https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-245.0.0-linux-x86_64.tar.gz?hl=ja
```

Comment:
Please add the page URL where the download link of gcloud is.
And divide the version string to an environment variable.

Reply:
The described Cloud SDK version differs between the Japanese page and the English page.
Add English page URL(hl=en) and set cloud SDK version which described in that page.

Reply:
Haven't changed `hl=ja`?

Reply:
No I have not changed this. (Already there at 2/26. I don't know why this is...)
But this parameter is unnecessary, so removed it.